packages/graphql/mixin.server.js

```diff
@@ -19,6 +23,24 @@ const fetch = require('cross-fetch');
 
 let introspectionResult = undefined;
 
+const renderMarkupAndHelmetFragments = async (element, canPrefetchOnServer) => {
+  if (canPrefetchOnServer) {
+    let helmetFragments = {};
+    const reactMarkup = await getMarkupFromTree({
+      tree: element,
+      renderFunction: tree => {
+        helmetFragments = renderStatic();
```

Comment:
I think you need to change the order of these two function calls. First `renderToString` and then `renderStatic`.
https://github.com/nfl/react-helmet#server-usage

Reply:
I'm using the untool's implementation now, so this is fixed.